estate/models/estate_property.py

```diff
@@ -0,0 +1,98 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, models, fields, _
```

Comment:
Imports must be in sorted manner.
```suggestion
from odoo import _, api, fields, models
```